### PR TITLE
Sphinx documentation and config added.

### PR DESCRIPTION
**Pull request by MissingNO57**

### CODE REVIEW

**Comment by panasevychol** [comment]
That is how homepage of the docs look like
image

File: Makefile [hunk 1]
```diff
@@ -141,6 +144,20 @@ liccheck:
 copyright-check:
 	python scripts/check_copyright.py
 
+####################
+### Docs generation
+####################
+
+.PHONY: docs
```
Comment (5A11) [inline Makefile:151]: change to `generate-docs`

File: Makefile [hunk 2]
```diff
@@ -19,9 +20,11 @@ else
     UNAME_S := $(shell uname -s)
     ifeq ($(UNAME_S),Linux)
         FIND_CMD := find $(COSMOS_PROTO_RELATIVE_DIRS) -regextype posix-extended
+				OPEN_CMD := xdg-open
     endif
     ifeq ($(UNAME_S),Darwin)
         FIND_CMD := find -E $(COSMOS_PROTO_RELATIVE_DIRS)
+				OPEN_CMD := open
```
Comment (5A11) [inline Makefile:27]: Add a windows version

**Review by OleBo — APPROVED** [review]
looks good to me

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**